src/main/java/org/example/DoWhileLoopMatrixGenerator.java

```diff
@@ -0,0 +1,31 @@
+package org.example;
+
+
+import static java.lang.Math.abs;
+
+public class DoWhileLoopMatrixGenerator {
+    static void generateMatrixWithDoWhileLoop(int[][] matrix, int n) {
```

Comment:
you can notice that each `generateMatrix...` method has a side effect - printing the matrix. They should be separated into different smaller methods.

Should be applied to all loop types

Reply:
I have added separate methods for printing the  matrix for each method